src/api/buddies.ts

```diff
@@ -44,10 +44,15 @@ export function fetchBuddies(
   );
 }
 
-const banRequest = (buddyId: string, accessToken: authApi.AccessToken) => {
+const banRequest = (
+  buddyId: string,
+  accessToken: authApi.AccessToken,
+  status: 'Active' | 'Banned',
```

Comment:
Yeah this active is bad word.

Reply:
for this we have now type BanStatuses which are 'Banned' | 'NotBanned'